manage.py

```diff
@@ -24,6 +24,7 @@ def main():
         GeventCommand('ptop_preindex'),
         GeventCommand('sync_prepare_couchdb_multi'),
         GeventCommand('sync_couch_views'),
+        GeventCommand('delete_old_couch_views_from_disk'),
```

Comment:
why does this need to use `gevent`?

Reply:
I didn't try too hard to find the root cause, but the thing that motivated me putting it here was that when it's not in this list, running `./manage.py delete_old_couch_views_from_disk` locally failed at the end with a gevent-related error. It's as if something in the internals of our couch library uses gevent directly and causes an issue if gevent patching hasn't already happened.

Reply:
Good enough for me